src/iota/transaction.py

```diff
@@ -45,6 +46,49 @@ def get_current_timestamp():
   return unix_timestamp(datetime.utcnow().timetuple())
 
 
+def convert_value_to_standard_unit(value, symbol='i'):
+  # type: (Text, Text -> float)
+  """
+    Converts between any two standard units of iota.
+
+    :param value:
+      Value (affixed) to convert. For example: '1.618 Mi'.
+
+    :param symbol:
+      Unit symbol of iota to convert to. For example: 'Gi'.
+
+    :return:
+      float as units of symbol to convert to
+  """
+  value_tuple = value.split()
+
+  try:
+    # Get input value
+    amount = float(value_tuple[0])
+  except (ValueError, IndexError) as e:
```

Comment:
Is `e` being used here?  Double-check your IDE's configuration; it should highlight this as an issue.